classes/form/CustomerAddressForm.php

```diff
@@ -42,6 +42,7 @@ public function __construct(
     public function loadAddressById($id_address)
     {
         $address = new Address($id_address, $this->language->id);
```

Comment:
how about ``$this->address = new Address($id_address, $this->language->id); `` ?